ParticlePhaseSpace/DataExporters.py

```diff
@@ -100,13 +101,19 @@ def _set_expected_units(self):
 
 class Topas_Exporter(_DataExportersBase):
     """
-    output the phase space to `topas ascii format <https://topas.readthedocs.io/en/latest/parameters/source/phasespace.html>`_.
+    output the phase space to `topas ascii or binary format <https://topas.readthedocs.io/en/latest/parameters/source/phasespace.html>`_.
```

Comment:
Could you add a note something like " you can switch between ascii or binary by passing the `binary` flag as a boolean, e.g. `binary=True` will generate a topas binary file"